ds3-autogen-java/src/test/java/com/spectralogic/ds3autogen/java/utils/TestGeneratedCode.java

```diff
@@ -50,12 +50,12 @@ public TestGeneratedCode(
 
     public void generateCode(
             final FileUtils fileUtils,
-            final String inputFileName) throws IOException, ParserException, ResponseTypeNotFoundException, TypeRenamingConflictException, TemplateModelException {
+            final String inputFileName) throws IOException, TemplateModelException {
         final Ds3SpecParser parser = new Ds3SpecParserImpl();
         final Ds3ApiSpec spec = parser.getSpec(TestGeneratedCode.class.getResourceAsStream(inputFileName));
         final CodeGenerator codeGenerator = new JavaCodeGenerator();
 
-        codeGenerator.generate(spec, fileUtils, Paths.get("."));
+        codeGenerator.generate(spec, fileUtils, Paths.get("."), null);
```

Comment:
I like the empty Ds3DocSpec even more, so we'll go with that in the next pull request.